### PR TITLE
Replace "sed json parser" to docker-machine inspect --format

### PR DESCRIPTION
**Pull request by labianchin**
`docker-machine inspect` already provide a way to get specific values of
the configuration. This removes the need of previous "(incomplete) json parser" with `sed`, simplifying the code.

### CODE REVIEW

File: src/docker-osx-dev [hunk 1]
```diff
@@ -381,27 +381,14 @@ function init_boot2docker {
...
-#
 # Configures variables based on the output of `docker-machine inspect $DOCKER_MACHINE_NAME`
 #
 function configure_docker_machine {
   DOCKER_HOST_NAME="$DOCKER_MACHINE_NAME"
-  DOCKER_MACHINE_INSPECT=$(docker-machine inspect "$DOCKER_MACHINE_NAME")
-  DOCKER_HOST_USER=$(echo -e "$DOCKER_MACHINE_INSPECT" | get_json_value "SSHUser")
-  DOCKER_HOST_IP=$(echo -e "$DOCKER_MACHINE_INSPECT" | get_json_value "IPAddress")
-  DOCKER_MACHINE_STORE_PATH=$(echo -e "$DOCKER_MACHINE_INSPECT" | get_json_value "StorePath")
+  DOCKER_HOST_USER=$(docker-machine inspect --format='{{.Driver.SSHUser}}' "$DOCKER_MACHINE_NAME" 2>&1)
```
Comment (brikis98) [inline src/docker-osx-dev]: Could you put these `docker-machine inspect` calls into a function (e.g. `get_docker_machine_config`) to reduce the repetition and make the code easier to read?

**Comment by brikis98** [comment]
Nice! I love it when code gets simpler. Just one minor comment and then this is good to go.

**Comment by labianchin** [comment]
Yes! Just did.

**Comment by brikis98** [comment]
Thanks!

**Pull request merged**